Author: Daniel Baumann <daniel@debian.org>
Description: Adjust to target directory.

diff -Naurp 9base.orig/rc/plan9ish.c 9base/rc/plan9ish.c
--- 9base.orig/rc/plan9ish.c	2009-08-05 08:10:47.000000000 +0000
+++ 9base/rc/plan9ish.c	2009-08-08 16:09:02.000000000 +0000
@@ -27,7 +27,7 @@ char *syssigname[]={
 char*
 Rcmain(void)
 {
-	return unsharp("#9/etc/rcmain");
+	return unsharp("/usr/lib/plan9/etc/rcmain");
 }
 
 char Fdprefix[]="/dev/fd/";
