Author: Daniel Baumann <daniel@debian.org>
Description: Adjust to target directory.

diff -Naurp 9base.orig/troff/font/devutf/mkMAP 9base/troff/font/devutf/mkMAP
--- 9base.orig/troff/font/devutf/mkMAP	2009-08-29 09:37:24.000000000 +0000
+++ 9base/troff/font/devutf/mkMAP	2009-09-15 17:17:52.000000000 +0000
@@ -1,4 +1,4 @@
-#!/bin/rc
+#!/usr/lib/plan9/bin/rc
 grep '	' $* | awk '
 $2 != "\"" {last = $4}
 {printf("\t%s\t%s\n", $1, last)}
diff -Naurp 9base.orig/troff/tmac/sendcover 9base/troff/tmac/sendcover
--- 9base.orig/troff/tmac/sendcover	2009-08-29 09:37:24.000000000 +0000
+++ 9base/troff/tmac/sendcover	2009-09-15 17:18:15.000000000 +0000
@@ -1,4 +1,4 @@
-#!/bin/rc
+#!/usr/lib/plan9/bin/rc
 T=/tmp/cs$pid
 cstrans $T $1
 rc $T
